emperor/template.py

```diff
@@ -0,0 +1,260 @@
+TEMPLATE_HTML = """
```

Comment:
Since these are templates, instead of having them as static strings, we can have them as files. Would you mind saving these as separate files i.e. one for `TEMPLATE_HTML` and one for `TEMPLATE_JS`, then put them in the support_files directory, and finally make sure that they are listed in the setup.py file?